brands/BOM/tokens/index.js

```diff
@@ -434,6 +434,17 @@ export default {
 		},
 	},
 
+	progressBar: {
+		borderWidth: '1px',
```

Comment:
I've been storing borderWidth values as ints (no px), then adding the 'px' back in the component (inside the template literal). That way we can remove the 'px' string in the token and be consistent with the other unit-less values, and do any unit-less calcs too.

Downside of this is that if the token value is ever updated to another unit it will break the component. Hmm 🤔

Reply:
We have to make choices in life... 🙂

Reply:
How very philosophical of you @florettewest

Reply:
I think it's best we do as you're doing and leave the px in the token